internal/btesting/btesting.go

```diff
@@ -178,6 +204,14 @@ func (db *DB) Fill(bucket []byte, numTx int, numKeysPerTx int,
 	return nil
 }
 
+// Convenience function for inserting a bunch of keys with 1000 byte values
+func (db *DB) FillWithKeys(numKeys int) error {
+	return db.Fill([]byte("data"), 1, numKeys,
+		func(tx int, k int) []byte { return []byte(fmt.Sprintf("%04d", k)) },
+		func(tx int, k int) []byte { return make([]byte, 1000) },
+	)
+}
```

Comment:
btesting.go provides some generic test utilities, such hard code (i.e. 1000 byte values) should test cases' responsibilities.
```suggestion
```

Reply:
ok